### ISSUE TITLE
LightAPI 1.13 ~ 1.19.4 Spigot & Paper & Hybrid. Download Here AND 1.20 Notice

### ISSUE THREAD

**Issue by IPECTER** [BeYkeRYkt/LightAPI#43]
Download: https://github.com/IPECTER/LighterAPI/releases/

@BeYkeRYkt If you're not going to stop developing LightAPI, check out code.

I think development of LightAPI was ended.
So Im developing new fork ([PhotonAPI](https://github.com/IPECTER/PhotonAPI)) (WIP)

**1.20 Notice**
Light Engine changed too many things in 1.20.
So, I need to learn it...
It will take long time... 2weeks 1months 3months maybe.
And one more. I will not update LighterAPI for 1_20_R2 and higher. (Photon API soon)

**Comment by qualk** [comment]
Thank you so much for this, I was just in the middle of trying to make it work for 1.19.4.

**Comment by lylesnake** [comment]
Hi, I know you are already aware of 1.20, but I got this error message when I tried to load your LightAPI and figured I would upload it to this comment in hopes that it may help you in your development. I'm very much looking forward to your update! 

19.06 23:53:54 [Server] [INFO] [LightAPI] Enabling LightAPI v5.4.0
19.06 23:53:54 [Server] [WARN] java.lang.ClassNotFoundException: ru.beykerykt.minecraft.lightapi.bukkit.internal.handler.craftbukkit.nms.v1_20_R1.StarlightNMSHandler
19.06 23:53:54 [Server] [WARN] at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:183)
19.06 23:53:54 [Server] [WARN] at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:150)
19.06 23:53:54 [Server] [WARN] at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
19.06 23:53:54 [Server] [WARN] at java.base/java.lang.Class.forName0(Native Method)
19.06 23:53:54 [Server] [WARN] at java.base/java.lang.Class.forName(Class.java:375)
19.06 23:53:54 [Server] [WARN] at LightAPI-Bukkit-5.5.0-SNAPSHOT.jar//ru.beykerykt.minecraft.lightapi.bukkit.internal.handler.craftbukkit.HandlerFactory.createHandler(HandlerFactory.java:83)
19.06 23:53:54 [Server] [WARN] at LightAPI-Bukkit-5.5.0-SNAPSHOT.jar//ru.beykerykt.minecraft.lightapi.bukkit.internal.BukkitPlatformImpl.initHandler(BukkitPlatformImpl.java:195)
19.06 23:53:54 [Server] [WARN] at LightAPI-Bukkit-5.5.0-SNAPSHOT.jar//ru.beykerykt.minecraft.lightapi.bukkit.internal.BukkitPlatformImpl.initialization(BukkitPlatformImpl.java:227)
19.06 23:53:54 [Server] [WARN] at LightAPI-Bukkit-5.5.0-SNAPSHOT.jar//ru.beykerykt.minecraft.lightapi.common.LightAPI.initialization(LightAPI.java:88)
19.06 23:53:54 [Server] [WARN] at LightAPI-Bukkit-5.5.0-SNAPSHOT.jar//ru.beykerykt.minecraft.lightapi.bukkit.BukkitPlugin.onEnable(BukkitPlugin.java:68)
19.06 23:53:54 [Server] [WARN] at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281)
19.06 23:53:54 [Server] [WARN] at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189)
19.06 23:53:54 [Server] [WARN] at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104)
19.06 23:53:54 [Server] [WARN] at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507)
19.06 23:53:54 [Server] [WARN] at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugin(CraftServer.java:562)
19.06 23:53:54 [Server] [WARN] at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugins(CraftServer.java:473)
19.06 23:53:54 [Server] [WARN] at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:636)
19.06 23:53:54 [Server] [WARN] at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:435)
19.06 23:53:54 [Server] [WARN] at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:308)
19.06 23:53:54 [Server] [WARN] at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1101)
19.06 23:53:54 [Server] [WARN] at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:318)
19.06 23:53:54 [Server] [WARN] at java.base/java.lang.Thread.run(Thread.java:833)
19.06 23:53:54 [Server] [WARN] java.lang.IllegalStateException: Initialization failed! Code: -1
19.06 23:53:54 [Server] [WARN] at LightAPI-Bukkit-5.5.0-SNAPSHOT.jar//ru.beykerykt.minecraft.lightapi.common.LightAPI.initialization(LightAPI.java:96)
19.06 23:53:54 [Server] [WARN] at LightAPI-Bukkit-5.5.0-SNAPSHOT.jar//ru.beykerykt.minecraft.lightapi.bukkit.BukkitPlugin.onEnable(BukkitPlugin.java:68)
19.06 23:53:54 [Server] [WARN] at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281)
19.06 23:53:54 [Server] [WARN] at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189)
19.06 23:53:54 [Server] [WARN] at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104)
19.06 23:53:54 [Server] [WARN] at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507)
19.06 23:53:54 [Server] [WARN] at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugin(CraftServer.java:562)
19.06 23:53:54 [Server] [WARN] at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugins(CraftServer.java:473)
19.06 23:53:54 [Server] [WARN] at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:636)
19.06 23:53:54 [Server] [WARN] at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:435)
19.06 23:53:54 [Server] [WARN] at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:308)
19.06 23:53:54 [Server] [WARN] at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1101)
19.06 23:53:54 [Server] [WARN] at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:318)
19.06 23:53:54 [Server] [WARN] at java.base/java.lang.Thread.run(Thread.java:833)
19.06 23:53:54 [Server] [INFO] [LightAPI] Disabling LightAPI v5.4.0
